finetune/nn/target_blocks.py

```diff
@@ -84,13 +83,12 @@ def masked_language_model(*, X, M, mlm_weights, mlm_positions, mlm_ids, embed_we
         mlm_ids = tf.reshape(mlm_ids, [-1])
         mlm_weights = tf.reshape(mlm_weights, [-1])
 
-        log_probs = tf.nn.log_softmax(logits, axis=-1)
-        one_hot_labels = tf.one_hot(mlm_ids, depth=n_vocab, dtype=tf.float32)
-        per_example_loss = -tf.reduce_sum(log_probs * one_hot_labels, axis=[-1])
```

Comment:
this mul by one_hot is pretty memory hungry but maybe a little bit faster. Need to run some more benchmarks.

Reply:
What was the output of these benchmarks?